### PR TITLE
This fixes a typo

### PR DESCRIPTION
**Pull request by gassmoeller**
@feathern: You should take a look at this.

### CODE REVIEW

**Comment by nicholasjnelson** [comment]
This is terrible.

File: INSTALL [hunk 1]
```diff
@@ -41,7 +41,7 @@ Rayleigh requires the the following compilers and libraries:
 
     OR
 
-3a. BLAS
+3a. BLAS.46
```
Comment (cianwilson) [inline INSTALL]: This looks like a typo.

**Comment by feathern** [comment]
Great!  But I think we're going another direction with these typo issues!

**Comment by tukss** [comment]
Shouldn't this be `.42`?

**Comment by gassmoeller** [comment]
> This is terrible.

Please keep comments friendly and constructive. E.g. write `please replace this by ...` or `In my mind this should look like this: ...`. This is a public repository and 26 users are currently following the discussions in this repository.

Comment to readers: We were having a in-person discussion about pull request review at the time of this PR, so all comments were friendly banter.

**Pull request closed without merging**